us.js

```diff
@@ -78,7 +80,13 @@
     }
     for (var i=0; i < us.STATES_AND_TERRITORIES.length; i++) {
       var state = us.STATES_AND_TERRITORIES[i];
-      if (val === state[field]) {
+      if (Array.isArray(state[field])) {
```

Comment:
added handling for values that are arrays, specifically for supporting states with multiple metaphones